nbgrader/docs/source/user_guide/release/ps2/problem3.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
Rather than creating two new assignments in the documentation, I think it might be cleaner to add some new example problems to the existing ps1. Could you remove ps2 and ps3 and instead add some example auto tests ps1?

Reply:
Ah right, I see, I missed ps3 was in R! Yes it's probably a good idea to showcase that. Though in that case I'd definitely make sure to link to ps3 (maybe rename it to ps2) from the docs so that it's clear that example exists.

Reply:
The source folder is now organized as:
ps1 including 3 Python notebooks (problem3.ipynb is a new notebook with autotest examples)
ps2 including 1 R notebook (problem1.ipynb)